cfme/fixtures/service_fixtures.py

```diff
@@ -34,16 +35,22 @@ def create_catalog_item(provider, provisioning, vm_name, dialog, catalog, consol
         logger.info("Console template name : {}".format(console_template.name))
         template = console_template.name
     item_name = dialog.label
-
-    provisioning_data = {
-        'catalog': {'vm_name': vm_name,
-                    },
-        'environment': {'host_name': {'name': host},
-                        'datastore_name': {'name': datastore},
-                        },
-        'network': {'vlan': partial_match(vlan),
-                    },
-    }
+    if provider.one_of(InfraProvider):
+        provisioning_data = {
+            'catalog': {'vm_name': vm_name},
+            'environment': {'host_name': {'name': host},
+                            'datastore_name': {'name': datastore}},
+            'network': {'vlan': partial_match(vlan)},
+        }
+    else:
+        provisioning_data = {
+            'catalog': {'vm_name': vm_name},
+            'properties': {'instance_type': partial_match(provisioning['instance_type']),
+                           'guest_keypair': provisioning['guest_keypair'],
+                           'boot_disk_size': provisioning['boot_disk_size']},
```

Comment:
REQUIRED: consider using  `provisioning.get('boot_disk_size', None)` - e.g. boot_disk_size is GCE specific and Azure doesn't have guest_keypair